ts/smelter/src/types/output/whip.ts

```diff
@@ -38,20 +57,23 @@ export type WhipAudioOptions = {
    */
   sendEosWhen?: OutputEndCondition | null;
   /**
-   * Audio encoder options.
+   * Audio encoder preferences list.
    */
-  encoder: WhipAudioEncoderOptions;
+  encoderPreferences?: WhipAudioEncoderOptions[] | null;
 };
 
-export type WhipAudioEncoderOptions = {
-  type: 'opus';
-  channels: Api.AudioChannels;
-  /**
-   * (**default="voip"**) Specifies preset for audio output encoder.
-   */
-  preset?: Api.OpusEncoderPreset;
-  /**
-   * (**default=`48000`**) Sample rate. Allowed values: [8000, 16000, 24000, 48000].
-   */
-  sampleRate?: number;
-};
+export type WhipAudioEncoderOptions =
+  | {
+      type: 'opus';
+      /**
+       * (**default="voip"**) Specifies preset for audio output encoder.
+       */
+      preset?: Api.OpusEncoderPreset;
+      /**
+       * (**default=`48000`**) Sample rate. Allowed values: [8000, 16000, 24000, 48000].
+       */
+      sampleRate?: number;
+    }
+  | {
+      type: 'any';
+    };
```

Comment:
I'm wondering whether something like this wouldn be better.
```suggestion
  | 'any';
```

any opinion on that?